packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+                            },
+                            program: 'spl-token',
+                        },
+                    }])
+                },
+            },
+        },
+    },
+});
+```
 
-Coming soon!
+## Querying Blocks
 
-#### Blocks
+Querying blocks is very similar to querying transactions, since a block
+contains a list of transactions. There's a bit more data at the highest level
+for a block, but you can query the list of transactions using a block query and
+transaction types in the same fashion you can query the lsit of accounts using
+a program accounts query and account types.
 
-Coming soon!
+```typescript
+const source = `
+    query myQuery($slot: BigInt!, $commitment: Commitment) {
+        block(slot: $slot, commitment: $commitment) {
+            blockHeight
+            blockhash
+            parentSlot
+            rewards {
+                //
+            }
+            transactions {
+                ... on TransactionJsonParsed {
+                    transaction {
+                        message {
+                            ... on TransactionMessageParsed {
+                                instructions {
+                                    ... on CreateAccountInstruction {
+                                        parsed {
+                                            info {
+                                                lamports
+                                                space
+                                            }
+                                            program
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
 
-#### Transactions
+const variableValues = {
+    slot: 43596n,
+    commitment: 'confirmed',
+};
+                                        
+const result = await rpcGraphQL.query(source, variableValues);
 
-Coming soon!
+expect(result).toMatchObject({
+    data: {
+        block: {
+            blockHeight: expect.any(BigInt),
+            blockhash: expect.any(String),
+            parentSlot: expect.any(BigInt),
+            rewards: expect.arrayContaining([{
+                commission: expect.any(Number),
+                lamports: expect.any(BigInt),
+                rewardType: expect.any(String),
+            }]),
+            transactions: expect.arrayContaining([{
+                message: {
+                    instructions: expect.arrayContaining([{
+                        parsed: {
+                            info: {
+                                lamports: expect.any(BigInt),
+                                space: expect.any(BigInt),
+                            },
+                            program: 'system',
+                        },
+                    }])
+                },
+            }]),
+        },
+    },
+});
+```
```

Comment:
Last time making the same output comment 😅

Reply:
Added!